### ISSUE TITLE
Incorrect checkstyle output on success

### ISSUE THREAD

**Issue by oliv3r** [hadolint/hadolint#986]
- [x] This is a bug report
- [ ] This is a feature request
- [ ] I searched existing issues before opening this one

### Expected behavior
When a file is properly linted, the tested (and passing) file should still be shown/included.

```xml
<?xml version='1.0' encoding='UTF-8'?>
<checkstyle version='4.3'>
</checkstyle>
```
 This is also important, and becomes an issue, when converting it to junit using an xslt, which then produces:

```xml
<?xml version="1.0" encoding="UTF-8"?>
<testsuite tests="0" failures="0"/>
```

which obviously is incorrect, as we pretend we ran zero tests, but we did run tests.

### Actual behavior
Comparing this to for example shellcheck's output, we see
```xml
<checkstyle version="4.3">
<file name="/builds/.../test/pass.sh"> </file>
</checkstyle>
```

Which then converts to
```xml
<testsuite tests="1" failures="0">
<testcase classname="/builds/.../test/pass.sh" name="/builds/.../pass.sh"> </testcase>
</testsuite>
```

Note that the number of tests.

### Steps to reproduce the behavior
`hadolint --format=checkstyle Dockerfile`

```bash
Haskell Dockerfile Linter 2.12.1
```

**Dockerfile** (if relevant)

```Dockerfile
FROM scratch
```

**Comment by m-ildefons** [comment]
Fixed in v2.15.0
Note that v2.15.0 also support native output in JUnit format

**Comment by oliv3r** [comment]
Did it in 2023? :) but thanks!!
